wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
+import socket
+
+import requests
+from requests import Session
```

Comment:
I find it a bit inconsistent to import the whole module and then import one object from that module. I'd rather do either:

1. Only `import requests`, then use `requests.Session` in your code below OR
2. Do `from requests import Session` and `from requests.exceptions import ConnectionError` and the use both of these (probably preferable, as it is consisten with importing `NewConnectionError` from `urllib3` (see next comment)